sporks.json

```
@@ -862,9 +862,55 @@
       }
     },
     "testnet": {
+      "testnet34": {
+        "id": 34,
+        "live": true,
+        "name": "Testnet34",
+        "sporkTime": "2022-04-04T16:00:00Z",
+        "rootHeight": "64904846",
+        "rootParentId": "0cbb13d9e7ed7092b5f20d0a20a79d1cae96fd796fb2bc569b27ce49cc97d97e",
+        "rootStateCommitment": "f2e77e16628543e285be8bac677db06e17b37f49e53a107af1e6bf99fbc17b30",
+        "gitCommitHash": "32bda868ae651cd792f7f78dee31911141c5da8e",
+        "stateArtefacts": {
+          "gcp": {
+            "rootCheckpointFile": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-34-execution/public-root-information/root.checkpoint",
+            "rootProtocolStateSnapshot": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-34/public-root-information/root-protocol-state-snapshot.json",
+            "rootProtocolStateSnapshotSignature": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-34/public-root-information/root-protocol-state-snapshot.json.asc",
+            "nodeInfo": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-34/public-root-information/node-infos.pub.json",
+            "executionStateBucket": "flow_public_devnet34_execution_state",
+            "protocolDBArchive": "https://storage.googleapis.com/flow-execution-archive/devnet34/protocol-db.tar.gz",
+            "protocolDBArchiveChecksum": "https://storage.googleapis.com/flow-execution-archive/devnet34/protocol-db.tar.gz.sha256",
+            "executionStateArchive": "https://storage.googleapis.com/flow-execution-archive/devnet34/execution-state.tar.gz",
+            "executionStateArchiveChecksum": "https://storage.googleapis.com/flow-execution-archive/devnet34/execution-state.tar.gz.sha256"
+          },
+          "s3": {
+            "rootCheckpointFile": "",
+            "rootProtocolStateSnapshot": "",
+            "nodeInfo": "",
+            "executionStateBucket": ""
+          }
+        },
+        "tags": {
+          "flow-go-tag": "v0.25.6",
+          "flow-dps-tag": "v0.24",
+          "cadence-tag": "v0.23.3",
+          "docker-tag": "v0.25.6"
+        },
+        "accessNodes": [],
+        "seedNodes": [
+          {
+            "address": "access-003.devnet34.nodes.onflow.org:3570",
+            "key": "b662102f4184fc1caeb2933cf87bba75cdd37758926584c0ce8a90549bb12ee0f9115111bbbb6acc2b889461208533369a91e8321eaf6bcb871a788ddd6bfbf7"
+          },
+          {
+            "address": "access-004.devnet34.nodes.onflow.org:3570",
+            "key": "0d1523612be854638b985fc658740fa55f009f3cd49b739961ab082dc91b178ed781ef5f66878613b4d34672039150abfd9c8cfdfe48c565bca053fa4db30bec"
+          }
+        ]
+      },
       "testnet33": {
         "id": 33,
-        "live": true,
+        "live": false,
         "name": "Testnet32",
```

while you're in here. can you update this
```suggestion
        "name": "Testnet33",
```